VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -29,20 +31,30 @@ final public class AuthenticationController
     let configuration: AppConfiguration
     let client: VimeoClient
     
+    /// We need to use a separate client to make the actual auth requests, to ensure it's unauthenticated
+    private let authenticatorClient: VimeoClient
```

Comment:
Does this mean that we can eliminate the client that's passed in to this class's constructor?